azure-pipelines/vs-insertion.yml

```diff
@@ -17,7 +17,7 @@ schedules:
     displayName: Daily VS insertion main
     branches:
       include:
-        - main
+        - vs17.14
```

Comment:
this would never apply, vs17.14 is inserted after each commit, either remove this or exclude from after-each commit (see line 32)